src/AudioToolbox/AudioType.cs

```diff
@@ -705,6 +717,17 @@ internal unsafe AudioChannelLayout (IntPtr h)
 			}
 		}
 
+		public AudioChannelLayoutNative NativeStruct {
+			get {
+				return new AudioChannelLayoutNative {
```

Comment:
The reason it's public is because the struct is used in CMFormatDescription.cs, making it private or internal it isn't accessible there.